doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -408,19 +408,19 @@ checks and is only meant to improve readability of the call.
     `wint_t`) on all platforms.
 
 !!! warning
-    A return type of `Union{}` means the function will not return i.e. C++11 `[[noreturn]]` or C11
+    A return type of `Union{}` means the function will not return i.e., C++11 `[[noreturn]]` or C11
```

Comment:
some punctuation is needed before `i.e.` as well, e.g. a semicolon.  Where a comma is required after "i.e." depends on the writing style.  The Julia manual is not internally consistent on this point.

Reply:
Added a comma before the i.e.